types/templates.go

```diff
@@ -428,6 +418,7 @@ type ValidatorPageData struct {
 	EstimatedNextWithdrawal                  template.HTML
 	AddValidatorWatchlistModal               *AddValidatorWatchlistModal
 	NextWithdrawalRow                        [][]interface{}
+	ValidatorPropsalData
```

Comment:
Out of curiosity I replaced
https://github.com/gobitfly/eth2-beaconchain-explorer/blob/34c75ca50b673dfa4b2c320ce17f05c9477e0fd0/handlers/validator.go#L417-L426
with
```
validatorPageData.ValidatorPropsalData = earnings.ProposalData
```
and it works. Please implement said change.

Reply:
that's neat ... I like it